fyle_accounting_mappings/models.py

```diff
@@ -538,3 +538,67 @@ def auto_map_ccc_employees(destination_type: str, default_ccc_account_id: str, w
                 )
 
         Mapping.objects.bulk_create(mapping_batch, batch_size=50)
+
+
+class EmployeeMapping(models.Model):
+    """
+    Employee Mappings
+    """
+    id = models.AutoField(primary_key=True)
+    source_employee = models.OneToOneField(ExpenseAttribute, on_delete=models.PROTECT)
+    destination_employee = models.ForeignKey(
+        DestinationAttribute, on_delete=models.PROTECT, null=True, related_name='destination_employee')
+    destination_vendor = models.ForeignKey(
+        DestinationAttribute, on_delete=models.PROTECT, null=True, related_name='destination_vendor')
+    destination_card_account = models.ForeignKey(
+        DestinationAttribute, on_delete=models.PROTECT, null=True, related_name='destination_card_account')
+    workspace = models.ForeignKey(Workspace, on_delete=models.PROTECT, help_text='Reference to Workspace model')
+    created_at = models.DateTimeField(auto_now_add=True, help_text='Created at datetime')
+    updated_at = models.DateTimeField(auto_now=True, help_text='Updated at datetime')
+
+    class Meta:
+        db_table = 'employee_mappings'
+
+    @staticmethod
+    def create_or_update_employee_mapping(
+            source_employee_id: int, workspace: Workspace,
+            destination_employee_id: int = None, destination_vendor_id: int = None,
+            destination_card_account_id: int = None):
+        """
+        Create single instance of employee mappings
+        :param source_employee_id: employee expense attribute id
+        :param workspace: workspace instance
+        :param destination_employee_id: employee destination attribute id
+        :param destination_vendor_id: vendor destination attribute id
+        :param destination_card_account_id: card destination attribute id
+        :return:
+        """
+        employee_mapping, _ = EmployeeMapping.objects.update_or_create(
+            source_employee_id=source_employee_id,
+            workspace=workspace,
+            defaults={
+                'destination_employee_id': destination_employee_id,
+                'destination_vendor_id': destination_vendor_id,
+                'destination_card_account_id': destination_card_account_id
+            }
+        )
+
+        return employee_mapping
```

Comment:
return employee_mapping, _
```suggestion
        return employee_mapping, _
```